cmd2/cmd2.py

```diff
@@ -2803,7 +2807,7 @@ def show(self, args: argparse.Namespace, parameter: str='') -> None:
         :param args: argparse parsed arguments from the set command
         :param parameter: optional search parameter
         """
-        param = parameter.strip().lower()
+        param = utils.norm_fold(parameter.strip())
```

Comment:
Good call.  We should have been doing proper unicode normalization and case folding